spec/factories/auctions.rb

```diff
@@ -116,6 +116,7 @@
     end
 
     trait :accepted do
+      ended_at { Time.now - 1.days }
```

Comment:
@jessieay I think also, as a rule, wouldn't an accepted auction always have an ended_at timestamp? And if so, shouldn't the factory reflect this?